### PR TITLE
[API Implementation] Get all products

### PR DESCRIPTION
**Pull request by dandalandone**
## Changes
- [API] Added Get all products
- Add sample API implementation

## Screenshot
image

## How to test?
- Run everything
- Go to http://localhost:5173/products then check if everything works.

### CODE REVIEW

**Review by TreshAnn — APPROVED** [review]
LGTM!

File: apps/web/src/views/products/pages/ProductsView.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import Product from 'ui/product/Product';
 
 import { useGetAllProducts } from '../api';
 import { CreateProductDTO, useCreateProduct } from '../api/addProduct';
+import { StyledContainer } from './styles';
+import { Grid } from '@mantine/core';
```
Comment (TreshAnn) [inline apps/web/src/views/products/pages/ProductsView.tsx]: duplicate import @mantine/core

Comment (sandrex-SGV) [reply apps/web/src/views/products/pages/ProductsView.tsx]: same issues

Comment (markjoshuacarmen) [reply apps/web/src/views/products/pages/ProductsView.tsx]: same same

**Review by sandrex-SGV — COMMENTED** [review]
works on my end, however, the product component seems to have different sizes maybe because of the image size (please see attached image)

image

**Comment by TreshAnn** [comment]
> works on my end, however, the product component seems to have different sizes maybe because of the image size (please see attached image)
> 
> image

and also the product prices must be displayed with 2 decimal places.

**Pull request merged**